components/AppBar.tsx

```diff
@@ -39,22 +36,22 @@ const AppBar = (props: AppBarProps) => {
     relevantAlerts =  alerts.filter(alert => alertInLocation(alert, {latitude:lat, longitude:lon}))
   }
 
-  return (
+  return useMemo(() => (
     <View style={styles.appBar}>
       <View style={styles.appTitleContainer}>
         {navigation.canGoBack() && 
         <TouchableOpacity accessible={true} accessibilityLabel='Go back' onPress={() => navigation.goBack()} style={{ paddingRight: 12 }}>
-            <Icon size={24} color='white' source={backArrow} />
+            <Icon size={28} color='white' source={backArrow} />
         </TouchableOpacity>}
-        <Text style={styles.appTitle} numberOfLines={1}>{props.location}</Text>
+        <Text style={styles.appTitle} numberOfLines={1}>{props.location || "Zanyengo"}</Text>
         {getWarningIcons(relevantAlerts)}
       </View>
 
       <View style={styles.appNav}>
         {showSearch && 
             <TouchableOpacity style={styles.items} accessible={true} accessibilityLabel='Search' 
-                onPress={() => router.navigate(SCREENS.Search.toString() as Href)}>
-                <Icon size={24} color='white' source="magnify" />
+                onPress={() => router.push(SCREENS.Search.toString() as Href)}>
+                <Icon size={28} color='white' source="magnify" />
```

Comment:
Increased icon size is to make it easier to tap the buttons?

Probably a good idea, but is it not possible to increase tappable area without increasing icon size? Could still be that the increased icon size is better though

Reply:
Yeah, we could alter the padding or something like that to increase the clickable area but I think increasing the side is better.